test/functional/wallet_send.py

```diff
@@ -305,6 +305,33 @@ def run_test(self):
         res = w2.walletprocesspsbt(res["psbt"])
         assert res["complete"]
 
+        # verify that fee estimation modes parse case insensitively
+        self.log.info("Testing case insensitive fee estimation mode parse")
+        for mode in ["ecoNOMICAL", "economical", "ECONOMICAL"]:
+            res = self.test_send(from_wallet=w0, to_wallet=w1, amount=1,
+                estimate_mode=mode, add_to_wallet=False
+            )
+            assert res["complete"]
+
+            res = self.test_send(from_wallet=w0, to_wallet=w1, amount=1,
+                arg_estimate_mode=mode, add_to_wallet=False
+            )
+            assert res["complete"]
```

Comment:
`estimatemode` is validated when passed with confirmation target
```suggestion
            res = self.test_send(from_wallet=w0, to_wallet=w1, amount=1,
                arg_conf_target=1, arg_estimate_mode=mode, add_to_wallet=False
            )
            assert_equal(res["complete"], True)
            
```